t/unit/backends/test_mongodb.py

```diff
@@ -5,14 +5,15 @@
 
 import pytest
 from case import ANY, MagicMock, Mock, mock, patch, sentinel, skip
+import bson
```

Comment:
is it in pypi?

Reply:
yep, seems like that - https://pypi.org/project/bson/. But I think that one that is used here goes from [pymongo](https://github.com/celery/celery/blob/master/requirements/extras/mongodb.txt#L1). I got inspiration from https://github.com/celery/celery/blob/master/celery/backends/mongodb.py#L24

Reply:
I've changed the import logic a bit to make pytest's discovery logic not to fail in case if pymongo/bson is missing.